src/environments/environment.dev.ts

```diff
@@ -0,0 +1,6 @@
+import { Environment } from './environment-interface'
+
+export const environment: Environment = {
+  production: true,
```

Comment:
This is the `environment.dev.ts`, shouldn't this be false?

Reply:
It should and it is now.